packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -10,7 +10,38 @@ Dark theme styles ship with PatternFly CSS by default. To enable dark theme, add
 
 Dark theme can also be applied based on the browser’s `prefers-color-scheme` media query. When using this method, we recommend using JavaScript to detect the media query and apply the `pf-[version]-theme-dark` class. This allows you to manipulate a single class in order to toggle dark theme based on user preference. [Adam Argyle wrote a great article](https://web.dev/building-a-theme-switch-component/#javascript) on using JavaScript to manage dark theme preference, using the `prefers-color-scheme` media query alongside a manual button that toggles dark theme. 
 
-[Our token resources](https://github.com/patternfly/patternfly/tree/v6/src/patternfly/base/tokens) include styles for both light and dark themes. When dark theme is enabled, your product will automatically pull dark theme tokens in order to adapt visual styles appropriately.
+[Our token resources](https://github.com/patternfly/patternfly/tree/v6/src/patternfly/base/tokens) include styles for both light and dark themes. When dark theme is enabled, your product will automatically pull dark theme tokens in order to adapt visual styles appropriately. 
+
+The only features which will require additional work in order to support dark theme are charts and images. Instructions for enabling support in these areas are included in the following sections.
+
+### Charts
+
+The [PatternFly React charts package](https://www.npmjs.com/package/@patternfly/react-charts) uses default style values that align with our light theme. In order to support dark-themed charts, you must import the stylesheet that contains dark theme styles by adding the following line to the top of your file:
+
+`import '@patternfly/patternfly/patternfly-charts.css';`
```

Comment:
This LGTM but I'll say I'm not sure if we have standard verbiage for how to import an asset like that? Specifically these 2 points:
* "adding the following line to the top of your file" - not sure if that's clear as to where it needs to go? I found this in the repo docs about [how to add PF CSS to your react app](https://github.com/patternfly/patternfly-react/tree/main/packages/react-core#project-configuration) - I'm assuming the charts CSS would be imported either in the same place or using similar language.
* This implies users have the `@patternfly/patternfly` package installed, which isn't needed for a react app by default, so not sure if it's worth calling that out? If so it would be similar to the linked page above in the [setup section](https://github.com/patternfly/patternfly-react/tree/main/packages/react-core#setup) with the `npm install` and `yarn add` commands.

Reply:
interesting points! I'll add a line for that first point similar to what you linked. For the second, wdyt about linking to [https://www.patternfly.org/get-started/develop#develop-with-htmlcss](https://www.patternfly.org/get-started/develop#develop-with-htmlcss)?